css-cascade/specificity.css

```diff
@@ -0,0 +1,19 @@
+* {
+  color: blue;
+}
+
+p {
+  color: orange;
+}
+
+.bio {
+  color: green;
+}
+
+#timbl {
+  color: red;
+}
+
+#verou {
+  color: black !important;
+}
```

Comment:
these selectors should be in reverse order to show off how specificity overrides source order